SU-SWS/ace-sulgryphon
# SUL23-154 | @jdwjdwjdw | Create taxonomy sets specific to Places to Study Content Type

# READY FOR REVIEW

# Summary
- Delete places to study map link, delete places to study accessibility taxonomy term
- Add Study Place Capacity term, and add capacity term select list
- Updated Places to Study form:
![image](https://user-images.githubusercontent.com/49299083/226496014-0f89de79-cd4c-4250-8b86-8b64ab3f51c5.png)

# Review By (Date)
- When convenient

# Steps to Test

1. Checkout branch
2. Add place to study and confirm the form fields look correct
3. Navigate to taxonomy and confirm that Study Place Capacity has been added
